packages/modeling/src/operations/extrusions/slice/calculatePlane.js

```diff
@@ -23,10 +23,13 @@ const calculatePlane = (slice) => {
   let farthestEdge
   let distance = 0
   edges.forEach((edge) => {
-    const d = vec3.squaredDistance(midpoint, edge[0])
-    if (d > distance) {
-      farthestEdge = edge
-      distance = d
+    // Make sure that the farthest edge is not a self-edge
```

Comment:
after repair (above), is this still possible?

Reply:
I fixed this case in two ways:

1. In `repairSlice`, since there is no need to have duplicate edges in a "repaired" slice. 
2. In `slice.calculatePlane` since in theory someone could call it from a different function, and then it would be nice if the plane calculation worked.

So to answer your question, no, this case can't be hit in `extrudeFromSlices` IF repair is enabled. If repair is not enabled, or `slice.calculatePlane` is called directly (it is a [publicly documented api](https://openjscad.xyz/docs/module-modeling_extrusions_slice.html)), then this is a nice fix, in my opinion.